ui/src/types/updates.types.ts

```diff
@@ -53,3 +55,16 @@ export enum UpdateStatus {
   UpToDate = 'up-to-date',
   UpdateClients = 'update-clients',
 }
+
+export interface GetChangelogItem {
```

Comment:
you can use camel case, I've also renamed the *Changelog* everywhere https://github.com/percona/grafana/pull/776/files#diff-336058a35246048f36019d0f034d8e56bfec5bfa5f32be6db5797821c456463a